test.php

```diff
@@ -0,0 +1,21 @@
+<?php
+require 'vendor/autoload.php';
+
+use Twig\Environment;
+use Twig\Loader\FilesystemLoader;
+
+// Initialize template engine https://twig.symfony.com/
+$loader = new FilesystemLoader('templates');
+$twig = new Environment($loader, [
+    'cache' => 'cache/twig',
+]);
+$num= $_GET['vala'];
+if(isset($a)) {
+    $i = 0;
+    $res = 0;
+    for (; $i < 11; $i++) {
+        $res = $i * $a;
+        echo " $i * $a = $res <br>";
```

Comment:
What is the reason to output here?

Reply:
I do not know how to show the results

Reply:
Pass variable to twig here https://github.com/HenintsoaHARINORO/ExoPHP/pull/6/files#diff-0524b271e915e8060f768ae50abd7ea860fa5ec3b9d842218c2c207347d77f1dR21
And use those variables in the template here 
https://github.com/HenintsoaHARINORO/ExoPHP/pull/6/files#diff-2292563b834bfb6a8fdf37eb126d1e22ad8df2e239268b0f2e9148ae099e49a3R3

Reply:
The first link did not show something